src/main/java/org/folio/inventory/dataimport/handlers/actions/AbstractInstanceEventHandler.java

```diff
@@ -48,7 +49,7 @@
   protected HttpClient client;
 
   protected Future<Void> createPrecedingSucceedingTitles(Instance instance, CollectionResourceRepository precedingSucceedingTitlesRepository) {
-    Future<Void> future = Future.future();
+    Promise<Void> future = Promise.promise();
```

Comment:
A promise should not be called future. Can you rename this variable and all similar changes in this pull request?
Learn how Future was split into Future and Promise: https://github.com/folio-org/raml-module-builder/blob/master/doc/futurisation.md